### ISSUE TITLE
error activating Mathjax plugin in WordPress

### ISSUE THREAD

**Issue by reakoh** [phillord/mathjax-latex#15]
Hello,

I'm new to using Mathjax and I'd like to try it on a wordpress.org webpage.
I downloaded the plugin but it gives me an error when I try to activate it:

Plugin could not be activated because it triggered a fatal error.
Parse error: syntax error, unexpected T_FUNCTION in /.../wp-content/plugins/mathjax-latex/mathjax-latex.php on line 235

Could somebody help with this please?

Kind Regards

**Comment by phillord** [comment]
Sorry for slow reply. Did you fix it?

**Comment by reakoh** [comment]
Yes thank you by replacing a couple of lines of code in the mathjax php file.

public static function filter_br_tags_on_math( $content ) {
    return preg_replace_callback(
        '/(<math.*>.*<\/math>)/isU',
        function ( $matches ) {
            return str_replace( array( '<br/>', '<br />', '<br>' ) , '' , $matches[0] );
        },
        $content
    );
}

**Comment by paulschreiber** [comment]
@reakoh What version of PHP are you using? What change did you make?
